app/controllers/application_controller.rb

```diff
@@ -27,6 +27,10 @@ def coronavirus_editor?
     current_user.has_permission? "Coronavirus editor"
   end
 
+  def redesinged_lists_permission?
```

Comment:
Tiny nit pick: there's a typo in this method name

```suggestion
  def redesigned_lists_permission?
```

Reply:
Good spot! Have fixed it per commit. Thanks 👍